mappings/net/minecraft/util/function/BooleanBiFunction.mapping

```diff
@@ -1,17 +1,34 @@
 CLASS net/minecraft/class_247 net/minecraft/util/function/BooleanBiFunction
+	COMMENT A function that is provided two booleans and returns one boolean.
 	FIELD field_1366 OR Lnet/minecraft/class_247;
+		COMMENT A {@link BooleanBiFunction} that returns {@code true} if either argument is {@code true}.
 	FIELD field_16885 NOT_OR Lnet/minecraft/class_247;
+		COMMENT A {@link BooleanBiFunction} that returns {@code true} if neither argument is {@code true}.
 	FIELD field_16886 ONLY_FIRST Lnet/minecraft/class_247;
+		COMMENT A {@link BooleanBiFunction} that returns {@code true} if only the first argument is {@code true}.
 	FIELD field_16887 SECOND Lnet/minecraft/class_247;
+		COMMENT A {@link BooleanBiFunction} that returns {@code true} if the second argument is {@code true}.
 	FIELD field_16888 NOT_AND Lnet/minecraft/class_247;
+		COMMENT A {@link BooleanBiFunction} that returns {@code true} if either argument is {@code false}.
 	FIELD field_16889 CAUSES Lnet/minecraft/class_247;
+		COMMENT A {@link BooleanBiFunction} that returns {@code true} if the first argument is {@code false} and the second argument is {@code true}.
 	FIELD field_16890 SAME Lnet/minecraft/class_247;
+		COMMENT A {@link BooleanBiFunction} that returns {@code true} if both arguments are {@code true} or both arguments are {@code false}.
 	FIELD field_16891 FIRST Lnet/minecraft/class_247;
+		COMMENT A {@link BooleanBiFunction} that returns {@code true} if the first argument is {@code true}.
```

Comment:
```suggestion
		COMMENT A {@link BooleanBiFunction} that returns the first argument.
```

Reply:
I think that it is more clear to state the output in terms of the input without explicitly rewriting the expression in sentence form. This may be applicable to other, more complex data types, but not booleans.

Reply:
If you want accuracy, should we use `returns ... if and only if` in place of all `returns ... if`?

Reply:
It could be `returns ... exclusively if` as well. I assumed all other cases returning `false` would be implied but I can clarify it if necessary.

Reply:
Then the current form should be good. Just make sure you fix the "causes" and "caused by" cases. a -> b is a causes b, or `!a || b` (like `a ? b : true`).